### ISSUE TITLE
Fork: Bitstream licence to OFL

### ISSUE THREAD

**Issue by raphaelbastide** [dejavu-fonts/dejavu-fonts#388]
We are working on a fork of one of the Dejavu font. We are wondering if can be possible to change the original license Bitstream to OFL? OFL not perfect, but it is well known by designers and libre font users now, and we think it can be more clear for them, what they can and can’t do with the font.

**Comment by alerque** [comment]
I don't think re-licensing the existing fonts is possible at this point. It would be theoretically possible if all the contributors to the Bitstream, Alef, and Dejavu projects collectively agreed to relicense all their past contributions but pragmatically speaking that's unlikely to happen.

That being said given the terms of the Bitstream license I think you can create a derivative work pretty easily. From a cursory review it should even be possible to create a fork and license the release of that fork as OFL, but you **also** need to abide by the terms of the Bitstream license which means you need to include a copy of its terms with your distributions.

**Comment by raphaelbastide** [comment]
> From a cursory review it should even be possible to create a fork and license the release of that fork as OFL

Yes, that’s exactly what we want to do, I realize now that my question was not clear enough. I don’t want to change Dejavu’s license, but just re-license our fork.

> but you also need to abide by the terms of the Bitstream license which means you need to include a copy of its terms with your distributions

You mean by dual licensing the fork, and include both OFL and Bitstream license textfiles?

**Comment by raphaelbastide** [comment]
Just so you know, with Jérémy Landes @StudioTriple, we made a fork of DejaVu sans that can spread! https://gitlab.com/velvetyne/fungal

**Comment by kimmus** [comment]
> Just so you know, with Jérémy Landes @StudioTriple, we made a fork of DejaVu sans that can spread! https://gitlab.com/velvetyne/fungal

404 :(

**Comment by moyogo** [comment]
Unfortunately this cannot happen unless Bitsteam and the authors of the other derivatives merged into DejaVu agree to change the license. Technically the DejaVu-only contributions are in public domain.

The first step would be to have Bitstream, now Monotype, change the Bitstream Vera fonts license to OFL.

**Comment by raphaelbastide** [comment]
@moyogo I didn’t know Bitstream is now Monotype. We ended by using the Bitstream License in the end, as it seems to be easier. But thank you for your clarifications.

@kimmus Sorry, it was piravate. It should be good now.

**Comment by raphaelbastide** [comment]
For your information, we will release the font [Fungal](https://velvetyne.fr/fonts/fungal/) today on Velvetyne Type Foundry. Specimen: https://fungal.page/

Thank you for your help! Thanks for DejaVu!
